app/models/bars/BarsCommon.scala

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.bars
+
+import cats.Eq
+import org.apache.commons.lang3.StringUtils
+import play.api.libs.json.{Format, Json}
+
+object BarsCommon {
```

Comment:
I think I'd find it easier to find things and see what's going on if the things inside here were in their own files. If you wanted more separation you could have a `request` and `response` package and then stick common stuff at the top level perhaps.